### DISCUSSION TITLE
set Action LinkColumn for Edit using wire:click

### DISCUSSION THREAD

**Discussion by priteshbarot** [rappasoft/laravel-livewire-tables#756] [Q&A] [answered]
LinkColumn::make('Action')
        ->title(fn($row) => 'Edit')
       ->location(fn($row) => '#' )
       ->attributes(fn($row) => [
              'class' => 'rounded-full',
              'wire:click' => 'edit('.$row->id.')',
      ]),

to call edit($id) method using livewire wire:click

**Comment by rappasoft** [comment, marked as answer]
Try `'wire:click.prevent' => 'edit('.$row->id.')',` to stop the default link behavior.
